src/EncoderBase64.h

```diff
@@ -9,8 +9,7 @@
 namespace cppkin
 {
     static const char* base64_padding[] = {"", "==","="};
-
-    inline std::string base64EncodeText(const std::string& data) {
+    static std::string base64EncodeText(const std::string& data) {
```

Comment:
Will create a duplicate impel under each translation unit, scoping it under class (helper class) and using static other there is better.